eng/Versions.props

```diff
@@ -45,7 +45,7 @@
     <MicrosoftCodeAnalysisVersion>3.8.0</MicrosoftCodeAnalysisVersion>
   </PropertyGroup>
   <PropertyGroup>
-    <MicrosoftCodeAnalysisNetAnalyzersVersion>6.0.0-preview6.21281.1</MicrosoftCodeAnalysisNetAnalyzersVersion>
+    <MicrosoftCodeAnalysisNetAnalyzersVersion>6.0.0-rc1.21320.2</MicrosoftCodeAnalysisNetAnalyzersVersion>
```

Comment:
Should we steadily update the version via dependency flow?

Reply:
Ideally,  yes.  Could you help with that?

This one's got a little more risk than most,  as new or updated rules could break the build and require some shepherding.  However, that should be rare.

Reply:
I pushed into your branch and added the dependency. Hope that's ok.

Reply:
That alone is insufficient, though, right?  Might be best as a standalone PR after this one goes in.

Reply:
That alone is sufficient as part of the repo work. I'm creating the subscription via the DARC tool right now.

Reply:
Really?  What about this hardcoded line we're commenting on here?

Reply:
This line gets updated as part of the "Update dependencies" Maestro PRs. The property already has the right name: PackageId without dots + the "Version" suffix. Maestro sees those and updates them automatically.

Reply:
Shouldn't it at a minimum be in the section below labeled `<!-- Arcade dependencies -->`?

Reply:
Yes, usually we add section headers. In this case it would be `<!-- roslyn-analyzers dependencies -->` above L48 but I wouldn't want to touch more files than necessary as this is your PR. We can add the comment either here or in a separate PR, whatever your prefer.

Reply:
Ok, thanks.